### DISCUSSION TITLE
hide product category

### DISCUSSION THREAD

**Discussion by aaronvt** [inventree/InvenTree#5558] [Q&A] [answered]
Is it possible to make a product category visible only to users in a certain role?

**Comment by SchrodingersGat** [comment, marked as answer]
@aaronvt no, this is currently not possible. It would require a reasonable amount of work "behind the scenes" to enable this sort of functionality in an efficient and error-free manner.

**Comment by aaronvt** [comment]
kind of what I figured, but thought it was worth asking, thanks.
